lib/tower_cli/resources/job.py

```diff
@@ -89,6 +90,8 @@ def launch(self, job_template=None, tags=None, monitor=False, timeout=None,
         # Get the job template from Ansible Tower.
         # This is used as the baseline for starting the job.
 
+        tags = kwargs.get('tags', None)
```

Comment:
I see what you're doing to maintain the same old use-pattern with tags here. But couldn't we remove _all_ the special cases for tags if we did a single replacement at the start where we pop `tags` from kwargs and put re-insert it as `job_tags`? I don't see any reason this should be different from limit or skip_tags, aside from a quick substitution of the alias.